src/ui/features/onboarding/Create/Create.tsx

```diff
@@ -10,20 +10,15 @@ import { PrivacyFooter } from 'src/ui/components/PrivacyFooter';
 import { wait } from 'src/shared/wait';
 import { accountPublicRPCPort, walletPort } from 'src/ui/shared/channels';
 import { zeroizeAfterSubmission } from 'src/ui/shared/zeroize-submission';
-import { useSizeStore } from '../useSizeStore';
+import { useSizeStore } from 'src/ui/shared/useSizeStore';
+import { isSessionExpiredError } from 'src/ui/shared/isSessionExpiredError';
+import { Stack } from 'src/ui/ui-kit/Stack';
 import { Password } from '../Password';
-import { Stack } from '../Stack';
 import { assertPasswordStep } from '../Password/passwordSearchParams';
 import * as helperStyles from '../shared/helperStyles.module.css';
-import { isSessionExpiredError } from '../shared/isSessionExpiredError';
+import { useOnboardingSession } from '../shared/useOnboardingSession';
 
-export function CreatePassword({
```

Comment:
Let's keep CreatePassword name here

Reply:
I renamed it because it creates a user (by creating a password and encryption key..., but these are internal details of the function we call). It literally calls the `createUser` function to create a user. So now, it matches the rest of the single-word component names like `Welcome`, `FAQ`, `Import`, etc. On the other hand, the `CreatePassword` name does match the page title

Reply:
Why not name it `CreateUser` then? Just `Create` is ambiguous

Reply:
ping

Reply:
updated ✅